frontend/tests/api/BaseApi.test.ts

```diff
@@ -76,4 +95,57 @@ describe("BaseApi", () => {
       }),
     );
   });
+
+  describe("Not OK response", () => {
+    let testApi: TestApi;
+
+    beforeEach(() => {
+      testApi = new TestApi();
+      global.fetch = jest.fn().mockResolvedValue({
+        json: () =>
+          Promise.resolve({
+            data: {},
+            errors: [],
+            message:
+              "The server could not verify that you are authorized to access the URL requested",
```

Comment:
Would this be user-facing in the frontend? Makes sense to say "you" to an API user, but might be confusing in the frontend. There, it's more that the app isn't authorized, not the end user.